bgcval2/bgcvaltools/makeMask.py

```diff
@@ -439,12 +442,12 @@ def makeMask(name, newSlice, xt, xz, xy, xx, xd, debug=False):
             'OffShelf',
     ]:
         #bathync = ncdfView("data/ORCA1bathy.nc",Quiet=True)
-        bathync = dataset("data/ORCA1bathy.nc", 'r')
+        bathync = dataset(os.path.join(paths.bgcval2_repo, "bgcval2/data/ORCA1bathy.nc"), 'r')
         bathy = abs(bathync.variables["bathymetry"][:])
         latcc, loncc = bathync.variables["lat"][:], bathync.variables["lon"][:]
         bathync.close()
-        shelfDepth = 500.
-        shelveFn = folder("shelves/MatchingMasks/") + "diag_maskMask.shelve"
+        shelfDepth = 250.
+        shelveFn = ukp.folder(os.path.join(paths.shelvedir, "MatchingMasks/"))+ "diag_maskMask.shelve"
```

Comment:
This is also one shelve file for everything. Though we can expect minor changes between model bathymetry (sea floor depth), in general it shouldn't be huge)

Reply:
this should be resolved now.